lib/webpacker/helper.rb

```diff
@@ -1,4 +1,11 @@
 module Webpacker::Helper
+  # Returns current Webpacker instance.
+  # Could be overriden to use multiple Webpacker
+  # configurations within the same app (e.g. with engines)
+  def current_webpacker
```

Comment:
I would just call `Webpacker.instance` directly instead of putting into a method. What do you think?

Reply:
The goal is to make it possible to override the Webpacker instance, so in the engine's helper we do:

```ruby
require "webpacker/helper"

module MyEngine
  module ApplicationHelper
    include ::Webpacker::Helper
    
    def current_webpacker
       MyEngine.webpacker # or whatever
    end
  end
end
```

`Webpacker.instance` would point to _global_ Webpacker (i.e. `::Webpacker.instance`), so that won't work